tap_zendesk/sync.py

```diff
@@ -28,7 +28,8 @@ def sync_stream(state, start_date, instance):
 
     parent_stream = stream
     with metrics.record_counter(stream.tap_stream_id) as counter:
-        for (stream, record) in instance.sync(state):
+        for (stream, record) in instance.sync(state) if instance.name != 'ticket_audits' \
```

Comment:
Can you put the iterator in a variable instead? this line is a bit tough to parse.

Reply:
@benekastah something like this?
```
to_process = instance.sync(state) if instance.name != 'ticket_audits' \
            else instance.sync(state, lookback_minutes)
        for (stream, record) in to_process:
```